cl/corpus_importer/management/commands/harvard_merge.py

```diff
@@ -543,11 +554,14 @@ def merge_overlapping_data(
                 )
             )
         elif field_name == "judges":
-            data_to_update.update(
-                merge_judges(
-                    changed_values_dictionary.get(field_name),
+            if skip_judge_merger:
+                logger.info(
+                    f"Can't merge judges, option disabled, cluster id: {cluster.id}"
+                )
+            else:
+                data_to_update.update(
+                    merge_judges(changed_values_dictionary.get(field_name))
```

Comment:
like here ... if we have skip judge merger we never try to merger judges - I would be more open to this flag - attempting to merge - and then we basically loosened the requirement.

Reply:
I pushed some changes, now, if the flag is set, merge won't be ignored, it will try to merge the judges, but if it fails, it will only log a message instead of raising an exception